res/skins/Tango/library.xml

```diff
@@ -7,8 +7,11 @@ Variables:
 -->
 <Template>
   <WidgetGroup>
+  <!-- Finally fixed: library can shrink to noting!
+      Solution was to define SizePolicy as 'xx,i'.
+      Found this in Deere-2.1 -->
     <ObjectName>Library</ObjectName>
-    <Size>1010me,74min</Size>
+    <SizePolicy>me,i</SizePolicy>
```

Comment:
Good find! I [documented this SizePolicy](http://mixxx.org/wiki/doku.php/creating_skins#sizepolicy) on the wiki.